src/sobol_sensitivity.jl

```diff
@@ -0,0 +1,109 @@
+function give_rand_y(f,p_range,p_fixed=nothing,indices=nothing)
+    if p_fixed == nothing
+        p = [rand(p_range[j][1]:1e-6:p_range[j][2]) for j in 1:length(p_range)]
```

Comment:
why are these choices discrete?

Reply:
I expected `rand(p_range[j][1]:p_range[j][2])` to work as a continuous interval but it wasn't correct, what would be the syntax for that?

Reply:
`(p_range[j][2]-p_range[j][1])*rand() + p_range[j][1]`